src/test/java/com/notably/logic/suggestion/model/ModelStub.java

```diff
@@ -0,0 +1,94 @@
+package com.notably.logic.suggestion.model;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import com.notably.commons.core.path.AbsolutePath;
+import com.notably.commons.core.path.exceptions.InvalidPathException;
+import com.notably.model.block.Block;
+import com.notably.model.block.BlockImpl;
+import com.notably.model.block.BlockTree;
+import com.notably.model.block.BlockTreeImpl;
+import com.notably.model.block.Title;
+import com.notably.model.suggestion.SuggestionItem;
+import com.notably.testutil.ModelStubBase;
+
+import javafx.beans.property.Property;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+/**
+ * A Stub class for Model.
+ */
+public class ModelStub extends ModelStubBase {
```

Comment:
Perhaps it's better if you move this stub to be a `private static class` of the `OpenSuggestionCommandTest` class. You're not using it any other place anyways. In addition, both of these classes are deeply coupled to one another, so perhaps separating them is not the best idea

Reply:
I also don't really like how this class is under the `com.notably.logic.suggestion.model` package. Seems rather unintuitive to have a `model` subpackage inside a `logic`